src/main/kotlin/com/exactpro/th2/codec/fixng/FixNgCodecSettings.kt

```diff
@@ -23,5 +23,6 @@ data class FixNgCodecSettings(
     val beginString: String = "FIXT.1.1",
     val dictionary: String,
     val charset: Charset = Charsets.US_ASCII,
-    val dirtyMode: Boolean = false
-) : IPipelineCodecSettings
+    val dirtyMode: Boolean = false,
+    val useStringValueFormat: Boolean = true
```

Comment:
suggestion: I would name it `decodeValuesAsString` instead. This parameter is used only during decoding, so, probably, it would be better to highlight that in the parameter name